src/tools/chdman.cpp

```diff
@@ -96,6 +97,7 @@ constexpr int MODE_GDI = 2;
 #define OPTION_INPUT "input"
 #define OPTION_OUTPUT "output"
 #define OPTION_OUTPUT_BIN "outputbin"
+#define OPTION_OUTPUT_SPLIT "split"
```

Comment:
I’d rather `--splitbin` and `-sb` or `--split` and `-s` so there’s more obvious correspondence between them.

Personally I’d lean towards `--splitbin` to hint that it affects the `--outputbin` file(s).

Reply:
Makes sense. Fixed: https://github.com/mamedev/mame/pull/12191/commits/118b7b2083b11b3d2d33488be26004ff73c6062a